unityfiles/Assets/Scripts/Helper/Util/WUGameObject.cs

```diff
@@ -0,0 +1,74 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+static class WUGameObjects{
+
```

Comment:
Declare if public or private, change name of the class to something more readable